### PR TITLE
feat: adding Opsgenie plugin DOCS

### PR DESCRIPTION
**Pull request by sandesvitor**
### ⚠️ &nbsp;&nbsp;Pre Checklist

> Please complete _ALL_ items in this checklist, and remove before submitting

- [x] I have `npm run build` and `npm run serve` locally before submitting this PR
- [x] I have read through the [Contributing](https://devlake.apache.org/community/) Documentation

# Summary

Adding Opsgenie plugin docs to both `next` and `v19`

### Does this close any open issues?
No

### Screenshots
Include any relevant screenshots here.

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

**Comment by d4x1** [comment]
LGTM.

File: versioned_docs/version-v0.19/Configuration/Opsgenie.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+title: "Opsgenie"
+sidebar_position: 11
+description: Config UI instruction for Opsgenie
+---
+
+Visit Config UI at: `http://localhost:4000`.
+
+## Step 1 - Add Data Connections
+
+On the Connections page, you can select Opsgenie and create a new connection or it.
```
Comment (d4x1) [inline versioned_docs/version-v0.19/Configuration/Opsgenie.md]: typo `or` ?  I think you mean `on`.

**Comment by d4x1** [comment]
If you add Opsgenie to doc v0.19,  make sure your codes have been cherry picked to branch release-v0.19.

**Comment by klesh** [comment]
Nice work!
However, the `v0.19` is not needed because it was already `beta`, which means we won't accept any new feature for the release.
The doc for the `next` is sufficient.

**Comment by sandesvitor** [comment]
Hey @d4x1 ! Thanks for the review. I think they will look into in v0.21 (https://github.com/apache/incubator-devlake/pull/6154). I will convert this to draft in the meantime =)

**Comment by sandesvitor** [comment]
Hey @d4x1 @klesh , sorry for the delay! I only left `next` documentation for Opsgenie and fixed the typo in Configuration.

File: versioned_docs/version-v0.19/Overview/SupportedDataSources.md [hunk 2]
```diff
@@ -9,83 +9,84 @@ sidebar_position: 5
...
+| Zentao (Beta)       | Issue Tracking                                                               | v17.x, v18.x                       | Advanced Mode Available | `zentao`                      | Full Refresh                                                        |
+| Gitee (WIP)         | Source Code Management, Code Review, Issue Tracking                          | Cloud                              | Not Available           | `gitee`, `gitextractor`       | Full Refresh, Incremental Sync(for `issues`,`MRs`)                  |
+| PagerDuty           | Issue Tracking                                                               | Cloud                              | Available               | `pagerduty`                   | Incremental Sync                                                    |
+| Opsgenie           | Issue Tracking                                                               | Cloud                              | Available               | `opsgenie`                     | Full Refresh (for `users`,`teams`), Incremental Sync (for `issues`) |
```
Comment (abeizn) [inline versioned_docs/version-v0.19/Overview/SupportedDataSources.md]: Hi @sandesvitor,  this may need to be deleted because this is version v0.19

Comment (sandesvitor) [reply versioned_docs/version-v0.19/Overview/SupportedDataSources.md]: Sorry! I'll updated it

Comment (sandesvitor) [reply versioned_docs/version-v0.19/Overview/SupportedDataSources.md]: Done! =)

**Review by abeizn — COMMENTED** [review]
lgtm

**Review by abeizn — APPROVED** [review]

**Pull request merged**